Ska/Sun.py

```diff
@@ -10,6 +10,51 @@
 
 __version__ = '3.5'
 
+# This is basically the SPM content of characteristics_general.  If the
+# flight version of the table is modified, this local copy may need updating.
+# Note that the version in characteristics basicaly specifies each "line segment"
+# with two rows of the table, allowing possibility for non-zero slope.  All of the
+# segments are flat, however, so this local copy does not bother with the duplication.
+# This ROLL_TABLE does not have all padding needed to provide 100% valid results
+# for planning purposes.
+ROLL_TABLE = Table.read(
```

Comment:
An inline table seemed easiest to manage, but this can obviously drift from the one in Matlab characteristics.